csp-tech-radar.csv

```diff
@@ -8,3 +8,40 @@ Long lived branches with Gitflow,hold,techniques,TRUE,"Gitflow is a strict branc
+Behave,hold,languages & frameworks,FALSE,"A tool for behavior-driven development (BDD) using tests written in a natural language style, backed up by Python code.<br/>Teams should try to favour BDD tests written with Spock test framework.<br /><br /><p><strong>Teams: </strong>Vesuvius, K2, Rosa</p>"
+Cucumber,hold,languages & frameworks,FALSE,"A software tool that supports behavior-driven development (BDD). Central to the Cucumber BDD approach is its ordinary language parser called Gherkin.<br />Teams should try to favour BDD tests written with Spock test framework.<br /><br /><p><strong>Teams: </strong>Gonzales, K2, Rosa, Fuji</p>"
+Cypress,assess,tools,FALSE,A purely JavaScript-based front-end testing tool.<br />Geb should be preferred instead.<br /><br /><p><strong>Teams: </strong>Gonzales</p>
+Gatling,adopt,tools,FALSE,"An open-source load testing solution for load testing and integrates with development pipelines. <br /><br /><p><strong>Teams: </strong>Wrights, Olympus, Krakatoa, Everest, Rockies, Gonzales, Vestrahorn, K2, Rosa, Fuji</p>"
+Geb,trial,tools,FALSE,"A browser automation solution which can be used for scripting, scraping and general automation. <br /><br /><p><strong>Teams: </strong>Olympus, Rockies, Gonzales, Vestrahorn</p>"
+Hamcrest,trial,languages & frameworks,FALSE,"A framework that assists writing tests in several langs. It supports creating customized assertion matchers, allowing match rules to be defined declaratively.<br /><br /><p><strong>Teams: </strong>Vesuvius, K2, Rosa</p>"
+Jest,adopt,languages & frameworks,FALSE,A JavaScript testing framework designed to ensure correctness of any JavaScript codebase. <br /><br /><p><strong>Teams: </strong>Gonzales</p>
+Junit,hold,languages & frameworks,FALSE,"A unit testing open-source framework for the Java. This refers to JUnit older than version 4<br /><br /><p><strong>Teams: </strong>Olympus, Gonzales, Huascaran , Annapurna, Fuji, K2, Rosa</p>"
+Mockito,adopt,languages & frameworks,FALSE,"A Java-based framework used for unit testing.  The framework allows the creation of test double objects in automated unit tests for the purpose of test-driven development or behavior-driven development. <br /><br /><p><strong>Teams: </strong>K2, Rosa</p>"
+Moto,trial,tools,FALSE,"A python library for Mocking AWS Services that we will be using to simulate AWS resources.<br /><br /><p><strong>Teams: </strong>K2, Rosa</p>"
+Pact,assess,tools,FALSE,"A code-first tool for testing HTTP and message integrations using contract tests.<br /><br /><p><strong>Teams: </strong>Olympus, Krakatoa, Everest, Fuji</p>"
+PyTest,adopt,languages & frameworks,FALSE,"The Python testing framework. It can be used to write various types of tests, including unit tests, integration tests, end-to-end tests, and functional tests. <br /><br /><p><strong>Teams: </strong>K2, Rosa</p>"
```

Comment:
```suggestion
PyTest,adopt,languages & frameworks,FALSE,"The Python testing framework. It can be used to write various types of tests, including unit tests, integration tests, end-to-end tests, and functional tests. <br /><br /><p><strong>Teams: </strong>K2, Rosa</p>"
```